libs/components/code-examples/src/lib/modules/text-editor/help-key/example.component.html

```diff
@@ -1,17 +1,15 @@
-<div class="sky-padding-even-xl">
```

Comment:
Removing these wrappers so that the demos don't have excessive padding on the docs site.